### PR TITLE
Fix a number of `igor` and `mandoc -Tascii -Tlint` errors

### LINKED ISSUE #167: kyua(1) has 2 spelling errors
From https://github.com/jmmv/kyua/blob/master/doc/kyua.1.in#L241 --
- expicitly should be explicitly.
- overriden should be overridden.

### PR DESCRIPTION
**Pull request by ngie-eign**
1. Always start new sentences on new lines.
2. Use more passive voice instead of "Let's...".
3. Fix some typos:
   i.   fileds -> fields
   ii.  file names -> filenames
   iii  arbitraty -> arbitrary
   iv.  plain-text -> plaintext
4. Add a missing "," before "respectively" to improve sentence flow.
5. kyua.1: Use `.Nm` when referring to "Kyua".
6. kyua-debug.1: clarify/wordsmith the description for
   `/dev/std{err,out}`.

Signed-off-by: Enji Cooper <yaneurabeya@gmail.com>

### CODE REVIEW

**Comment by ngie-eign** [comment]
CC: @0mp

File: doc/kyua.1.in [hunk 1]
```diff
@@ -62,15 +62,19 @@ simple fix to a library to the system administrator deploying a new
 release on a production machine.
 .Pp
 Kyua is able to execute test programs written with a plethora of testing
-libraries and languages.  The library of choice is ATF, for which Kyua
-was originally designed, but simple, framework-less test programs and
-TAP-compliant test programs can also be executed through Kyua.
+libraries and languages.
+The test program library of choice is ATF, which
+.Nm 's
```
Comment (0mp) [inline doc/kyua.1.in]: It does not work as you thought it works :) It only prints `'s` now.

**Review by 0mp — CHANGES_REQUESTED** [review]
Great work overall.

File: doc/kyua.1.in (hunk 1 above)
Comment (ngie-eign) [reply doc/kyua.1.in]: That's unfortunate :/.. I'll restore that hardcoded reference.

Comment (0mp) [reply doc/kyua.1.in]: Try `.Nm Ns 's` instead ;)

File: doc/kyua-debug.1.in [hunk 2]
```diff
@@ -25,7 +25,7 @@
 .\" THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 .\" (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-.Dd October 13, 2014
+.Dd February 28, 2019
```
Comment (jmmv) [inline doc/kyua-debug.1.in]: I don't think the date should change if there were no content changes.

File: doc/kyua.1.in [hunk 3]
```diff
@@ -25,7 +25,7 @@
 .\" THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 .\" (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 .\" OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-.Dd May 12, 2015
+.Dd February 28, 2019
```
Comment (jmmv) [inline doc/kyua.1.in]: nit: Same here regarding the date.

File: doc/kyua.1.in [hunk 4]
```diff
@@ -220,29 +227,34 @@ And the default naming scheme of the log files is:
...
-problems in the application, as they contain lots of internal details.
+Detailed informational messages.
+These are often useful when debugging problems in the application, as they
+contain lots of internal details.
 .El
 .Pp
 The default log level is
 .Sq info
-unless expicitly overriden with
+unless explicitly overridden with
```
Comment (jmmv) [inline doc/kyua.1.in:253]: Oops... you'll get a merge conflict here.

File: doc/build-root.mdoc [hunk 5]
```diff
@@ -31,9 +31,9 @@ the concept that allows a developer to keep the source tree clean from
 build products by asking the build system to place such build products
 under a separate subtree.
 .Pp
-Most build systems today support build directories.  For example, the GNU
-Automake/Autoconf build system exposes such concept when invoked as
-follows:
+Most build systems today support build directories.
+For example, the GNU Automake/Autoconf build system exposes such concept when
```
Comment (jmmv) [inline doc/build-root.mdoc:35]: Note that the existing text used two spaces after a period which should have done the right thing when rendered (i.e. this should have acted the same way as a newline).

I understand linting tools suggest newline at end of sentence, so OK.

**Comment by ngie-eign** [comment]
@jmmv: done!

File: doc/kyua.1.in (hunk 1 above)
Comment (jmmv) [reply doc/kyua.1.in]: I don't know if the GitHub UI is tricking me, but this doesn't seem resolved.

File: doc/build-root.mdoc (hunk 5 above)
Comment (ngie-eign) [reply doc/build-root.mdoc:35]: For whatever reason, line breaks are the defacto linting rule implemented by the mandoc project 🤷‍♀️.

File: doc/kyua.1.in (hunk 1 above)
Comment (ngie-eign) [reply doc/kyua.1.in]: > I don't know if the GitHub UI is tricking me, but this doesn't seem resolved.

I forgot to handle it before, but it's fixed properly now.

**Pull request merged**